tests/berlin/eip2929_gas_cost_increases/test_gas_cost.py

```diff
@@ -0,0 +1,1071 @@
+"""Tests supported precompiled contracts."""
+
+import pytest
+
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+from ethereum_test_vm.bytecode import Bytecode
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-2929.md"
+REFERENCE_SPEC_VERSION = "0e11417265a623adb680c527b15d0cb6701b870b"
+
+UPPER_BOUND = 0xFF
+NUM_UNSUPPORTED_PRECOMPILES = 8
+
+
+@pytest.fixture
+def contract_bytecode(request) -> Bytecode:
+    """Generate bytecode for the contract to be tested."""
+    return (
+        Op.GAS
+        + Op.CALL(
+            address=request.param["address"],
+            value=request.param["value"],
+            args_offset=request.param["args_offset"],
+            args_size=request.param["args_size"],
+            output_offset=request.param["output_offset"],
+            output_size=request.param["output_size"],
+        )
+        + Op.POP
+        + Op.SUB(Op.SWAP1, Op.GAS)
+        + Op.GAS
+        + Op.CALL(
+            address=request.param["address"],
+            value=request.param["value"],
+            args_offset=request.param["args_offset"],
+            args_size=request.param["args_size"],
+            output_offset=request.param["output_offset"],
+            output_size=request.param["output_size"],
+        )
+        + Op.POP
+        + Op.SUB(Op.SWAP1, Op.GAS)
+        + Op.SWAP1
+        + Op.SSTORE(0, Op.SUB)
+        + Op.STOP
+    )
+
+
+def initial_setup_bytecode(sender: Account) -> Bytecode:
+    """Generate bytecode for the initial setup to be tested."""
+    return (
+        Op.MSTORE(0, 0)
+        + Op.MSTORE(0x100, 0xDEADBEEF)
+        + Op.CALL(
+            address=sender,
+            value=1,
+            args_offset=0,
```

Comment:
i think this are the default values anyway

Reply:
I'll omit the zero value arguments to use defaults 👍